backend/de.metas.payment.sepa/base/src/main/java/de/metas/payment/sepa/sepamarshaller/impl/SEPAVendorCreditTransferMarshaler_Pain_001_001_03_CH_02.java

```diff
@@ -246,15 +246,10 @@ public void marshal(
 			@NonNull final I_SEPA_Export sepaDocument,
 			@NonNull final OutputStream out)
 	{
-		try
-		{
-			final Document xmlDocument = createDocument(sepaDocument);
-			marshal(xmlDocument, out);
-		}
-		catch (final Exception e)
-		{
-			throw new AdempiereException("Error while marshaling " + sepaDocument, e);
-		}
+
+		final Document xmlDocument = createDocument(sepaDocument);
+		marshal(xmlDocument, out);
```

Comment:
why we are no longer catching the exception here? what's the reason behind?
might be ok, but i want to understand.

Reply:
![image](https://user-images.githubusercontent.com/19719799/182387568-cd5804e7-f624-430d-93ee-24864320d0b8.png)

The error messages where not displayed correctly.